### ISSUE TITLE
Project Plan Update Mobile

### ISSUE THREAD

**Issue by selinisik** [bounswe/bounswe2023group4#216]
### Problem Definition

During the lecture, our instructor suggested adding extra milestones to the project plan for more frequent progress evaluations. Thus, we will determine where to insert these additional milestones in the mobile program.

**Comment by YigitSekerci** [comment]
I have fixed the milestone times and added some review tasks near the milestone 3 however we didn't determine the extra milestone times yet I'll wait for the extra milestones then I'll update the wiki.

**Comment by YigitSekerci** [comment]
I have updated the project plan in [this wiki page](https://github.com/bounswe/bounswe2023group4/wiki/Mobile-Project-Plan). We have introduced 2 new milestones. I have divided the tasks for deliverables. I have tried to wait others' plan to arrange mobile plan accordingly however I think they also waiting others to finish thus I have finished the mobile plan. Naturally plan may change in following weeks but these deliverables looks good in my opinion.
